setup.sh

```diff
@@ -123,13 +123,17 @@ if linux; then
 fi
 
 if ! hash gdb; then
-    echo 'Could not find gdb in $PATH'
+    echo "Could not find gdb in $PATH"
     exit
 fi
 
 # Update all submodules
 git submodule update --init --recursive
 
+# Set Python3 as the default.
+rm -f /usr/bin/python
+ln -s /usr/bin/python3 /usr/bin/python
```

Comment:
The Docker container not started with the Python binary, which cause a failure but most important also in the GitHub workflows it takes Python v2 rather than the v3, it's important to set the standard Python to v3, just in case.

```text
[Unit tests/tests] ⭐  Run Python version info
  echo 'GDB py:'
  gdb --batch --quiet --nx --nh --ex 'py import sys; print(sys.version)'
  echo 'Installed py:'
  python -V
GDB py: 3.6.9 (default, Oct  8 2020, 12:12:24)  [GCC 8.4.0]
Installed py: Python 2.7.17
```